dts/arm/nxp/nxp_mcxn94x_common.dtsi

```diff
@@ -918,6 +918,17 @@
 			status = "disabled";
 		};
 	};
+
+	rtc: rtc@4c000 {
+		compatible = "nxp,irtc";
+		reg = <0x4c000 0x1000>;
+		status = "disabled";
+		interrupts = <52 0>;
+		prescaler = <1>;
+		clock-frequency = <16000>;
```

Comment:
is the clock-frequency related to the clock-select?

Reply:
Yes it is.

Reply:
don't you need to document it?

Reply:
There is a reference to it in the rtc.yaml

Reply:
but you just said youre using it in a way specific to this irtc ?